impl/client/src/test/javascript/util/xss.spec.js

```diff
@@ -76,6 +76,19 @@ define(["common-ui/util/xss"], function(xssUtil) {
           xssUtil.setHtml(elem, unsafeHtml);
           expect(elem.innerHTML).not.toContain("<script>");
         });
+
+        it("should set sanitized HTML to a jQuery object", function () {
+          const elem = $("<div></div>");
```

Comment:
Should there not be a `"common-ui/jquery"` dependency in this (test) module, with parameter `$`?